container_api/src/holochain.rs

```diff
@@ -88,7 +90,12 @@ impl Holochain {
         instance.start_action_loop(context.clone());
         let context = instance.initialize_context(context.clone());
         let context2 = context.clone();
-        let result = block_on(initialize_application(dna, &context2));
+        let result = block_on(
```

Comment:
Yeah, I was hesitant to change the Rust edition and feature flags in another create (container_api) just for this because I regard these few lines as temporary anyway since I want to extract that into an initialization workflow...